### PR TITLE
extract redirectUser method

### PR DESCRIPTION
**Pull request by pabloleonalcaide**
extract a 'redirectUser' method to avoid duplicated code

### CODE REVIEW

**Comment by benedmunds** [comment]
Can you resolve the conflicts and resubmit (likely need to pull 3 in to your branch).

**Comment by pabloleonalcaide** [comment]
Hi, I think that now there are no conflicts, sorry!

File: controllers/Auth.php [hunk 1]
```diff
@@ -853,17 +848,12 @@ public function _get_csrf_nonce()
 	/**
 	 * @return bool Whether the posted CSRF token matches
 	 */
-	public function _valid_csrf_nonce()
-	{
+	public function _valid_csrf_nonce(){
 		$csrfkey = $this->input->post($this->session->flashdata('csrfkey'));
-		if ($csrfkey && $csrfkey === $this->session->flashdata('csrfvalue'))
-		{
+		if ($csrfkey && $csrfkey === $this->session->flashdata('csrfvalue')){
```
Comment (benedmunds) [inline controllers/Auth.php]: Move the curly brace to the next line to match other styling.

**Review by benedmunds — CHANGES_REQUESTED** [review]

**Comment by benedmunds** [comment]
Thanks! Looks good, merging.

**Pull request merged**